pkg/multus/multus.go

```diff
@@ -536,17 +553,17 @@ func CmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) (c
 		pod, err = kubeClient.GetPod(string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_NAME))
 		if err != nil {
 			var waitErr error
-			// in case of ServiceUnavailable, retry 10 times with 0.5 sec interval
-			if errors.IsServiceUnavailable(err) {
-				pollDuration := 500 * time.Millisecond
+			// in case of a retriable error, retry for 5 seconds, with exponential backoff
+			if isCriticalRequestRetriable(err) {
+				pollDuration := 250 * time.Millisecond
 				pollTimeout := 5 * time.Second
-				waitErr = wait.PollImmediate(pollDuration, pollTimeout, func() (bool, error) {
+				waitErr = wait.ExponentialBackoff(generateRetryBackoff(pollDuration, pollTimeout), func() (bool, error) {
```

Comment:
@dougbtv What do you think about to change to ExponentialBackoff()?

Reply:
I think we need pollimmediate here too, it's so we can keep retrying quickly during install/upgrade so we can retry when other components are in flux.

Reply:
OK, I had misunderstood the issue - using a smaller retry frequency, with a smaller timeout (50% less).